src/content/developers/docs/intro-to-ether/index.md

```diff
@@ -0,0 +1,91 @@
+
+It is [common](https://www.reuters.com/article/us-crypto-currencies-lending-insight-idUSKBN25M0GP#:~:text=price%20of%20ethereum) [to](https://abcnews.go.com/Business/bitcoin-slumps-week-low-amid-renewed-worries-chinese/story?id=78399845#:~:text=cryptocurrencies%20including%20ethereum) [conflate](https://www.cnn.com/2021/03/14/tech/nft-art-buying/index.html#:~:text=price%20of%20ethereum) Ethereum and Ether — when people reference the "price of Ethereum," they are almost always describing the price of Ether.
+
+## Pricing a Transaction in Ether {#pricing-a-transaction}
+
+The price of a transaction on Ethereum is determined by two variables: gas and gas fees. By multiplying these variables, users can determine the price of their transaction — which is paid in Ether.
+
+[Gas](/developers/docs/gas/#what-is-gas) is proportional to the number of operations a given transaction must execute before completing. Some transactions might be very simple; for example, moving Ether from one account to another. Other transactions are much more intensive; for example, calculating the first trillion Fibonacci numbers. Tracking gas ensures each user pays a proportional amount for their usage of the Ethereum network's computing power.
+
+Gas fees are a measure of the short-term demand for the Ethereum network's computing power. When many users are trying to use Ethereum, gas fees increase (and vice versa).
+
+Gas fees are made up of two components: [base fee](/developers/docs/gas/#base-fee) and [priority fee](/developers/docs/gas/#priority-fee) (also called a tip). The base fee is set by the protocol in reaction to network demand, and the priority fee is set by users hoping to be included into the next block. Senders can set their priority fee as high as they prefer; however, the miners who include transactions into blocks are incentivized to pick transactions offering the highest gas fees.
+
+Thus, gas fees functions as an auction mechanism for inclusion in the next Ethereum block. As a result, two transactions with the same amount of gas may cost more or less depending on when they are submitted to Ethereum.
+
+[More on gas](/developers/docs/gas/)
+
+## Minting Ether {#minting-ether}
+
+Ether is minted by the Ethereum protocol when a miner creates a block. To incentivize block creation, the protocol grants a "reward" in each block, incrementing the balance of an address set by the block's miner. The block reward has changed over time, and today it is 2 ETH per block.
+
+Ethereum also provides smaller rewards to miners who create ["uncle blocks"](/developers/docs/data-and-analytics/block-explorers/#blocks), which are beyond the scope of this introduction.
```

Comment:
Confirming: Remove just the uncle blocks paragraph, or both the miner rewards and uncle blocks paragraphs? Seeing both highlighted in the Github comment. 

Agreed on uncle blocks (too confusing for intro), but think we may want to keep the miner reward paragraph because "where does ETH come from?" feels like an intro-level question we should answer.

Reply:
Yeah, I meant the uncle block section. The section above is good.